config/scheduler/email.template

```diff
@@ -0,0 +1,16 @@
+Dear ProActive User,
+
+This notification is to inform you that the ProActive Job ID: %1$s is %2$s.
+
+Below, you find more details about the job tasks and their statuses:
+%3$s
```

Comment:
can you not use named patterns that you could replace?
For example %JOB_ID%, etc?

Reply:
Done. Check it out here https://github.com/ow2-proactive/scheduling/pull/3702/commits/047b89c3e172dd503f55e95dfd88d6abaa0f1f39